aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/UpdateHandler.java

```diff
@@ -40,12 +41,21 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(final Amaz
                 .resourceTags(Translator.translateTagsToSdk(request.getDesiredResourceState().getTags()))
                 .build();
 
+        final boolean shouldUpdateParameters = !DifferenceUtils.diff(request.getPreviousResourceState().getParameters(), request.getDesiredResourceState().getParameters()).isEmpty();
```

Comment:
I am afraid this will miss out of band changes.

Reply:
I think that benefit of avoiding an interruption of service to DBClusters which are affected by changes to DBClusterParameterGroup outweighs the support of DBClusterParameterGroup out of bounds updates. Out of band stack updates are strongly discouraged by CloudFormation.